janusgraph-core/src/main/java/org/janusgraph/core/attribute/Geoshape.java

```diff
@@ -554,14 +554,14 @@ public Geoshape read(ScanBuffer buffer) {
             long l = VariableLong.readPositive(buffer);
             assert l>0 && l<Integer.MAX_VALUE;
             int length = (int)l;
+            int position = ((ReadArrayBuffer) buffer).getPosition();
```

Comment:
AFAICT this variable isn't used anywhere other than on L#564; should it be moved into the inner scope block?